django_project/core/settings/base.py

```diff
@@ -133,3 +133,6 @@
 
 UPLOADER['SUPPORTED_CRS'].append('ESRI:54030')
 EPSG_CODE_MATCHES['ESRI:54030'] = '(54030) World_Robinson'
+
+# Where the file uploaded temporary saved
+FILE_UPLOAD_TEMP_DIR = os.getenv('TEMP_FILE_UPLOAD_ROOT', None)
```

Comment:
It would hep if both the django settings and the environment file uses the same name or prefix+same name.
Also, wouldn't you need to set default value? I'm not sure where the setting is going to be used.

Reply:
default value is None (checked on the geonode)
when none, it will use '/temp/'

it is used when we upload shapefile, where it will be uploaded
currently, it will be uploaded to '/temp', because the default is None
Geonode has multiple steps process to upload shapefile (multiple GET)
But because it's uploaded to '/temp' when we upload shapefile as layer, when the next steps is called, it will not found to another container if we scale container more than 1

Use case:
- POST : User upload shapefile to container 1, save it to temp
- GET : Step check from geonode, it will ask for example to container 2, but it does not exist

Reply:
the key name in env has been updated